notebooks/pythonjobtemplate.ipynb

```diff
@@ -59,7 +58,8 @@
     "job = pr.create_job(job_type=ToyJob, job_name=\"toy\")\n",
     "print(job.input)\n",
     "job.run()\n",
-    "job['output/generic/energy_tot']"
+    "#job['output/generic/energy_tot']\n",
+    "job[\"output/data\"][\"energy_tot\"]"
```

Comment:
The output should be accessible using: `job["output/generic/energy_tot"]` as that is the unique path we commonly use, in particular it should not be separated in two parts like `job["output/data"]["energy_tot"]`